js-miniapp-sdk/src/miniapp.ts

```diff
@@ -35,6 +36,12 @@ interface MiniAppFeatures {
    * @returns The Promise of share info action state from injected side.
    */
   shareInfo(info: ShareInfoType): Promise<string>;
+
+  /**
+   * @param screenAction The action that miniapp wants to request on device.
+   * @returns The Promise of screen action state from injected side.
+   */
+  requestScreenOrientation(screenAction: ScreenAction): Promise<string>;
```

Comment:
`requestScreenOrientation` sounds like you just want to retrieve what the current orientation is. 

How about `setScreenOrientation` or  `requestScreenOrientationLock` instead?